tests/validation/tests/v1_api/test_deployment.py

```diff
@@ -0,0 +1,81 @@
+from .common import *  # NOQA
+import pytest
+
+namespace = {"client": None, "ns": None}
+
+
+def test_namespace_create():
+    t = {
```

Comment:
this is a small payload but shouldn't it be placed in the resource dir too?

Reply:
I'd rather leave it here as a demonstration of generating a small payload in the test method directly.

Reply:
I'm still not quite sure of leaving it inline as we're not going to be consistent but if it'll help for demonstrative purposes it's ok

Reply:
@izaac pointed out that I missed this. 

A couple of notes here: 

An inline declaration of the resource is fine if it's a small resource and used in support of another workflow; but if we are explicitly testing the resource we should have a template to reference in the test. 

As we are testing the namespace creation here we should:
1) Create a template for the namespace resource
2) Move the test_namespace test to its own file as it doesn't make sense to tie it to the deployment tests.

Reply:
Actually, I have switched all of them to use the template. 
Either way is ok for me.